hdk/virtual_machine/parser.py

```diff
@@ -0,0 +1,82 @@
+"""Functions for parsing symbolic instructions into their underlying fields."""
+import re
+
+from hdk.virtual_machine.syntax import (
+    Instruction,
+    MemoryAccessInstruction,
+    StackInstruction,
+)
+
+
+def remove_comment(line: str) -> str:
```

Comment:
Remove this function. You don't need it. In VM language only comment lines starting with // are allowed. In such cases the whole line is a comment. Comments in other places are not allowed.